pkg/provisionserver/configmap.go

```diff
@@ -0,0 +1,34 @@
+package provisionserver
+
+import (
+	ospdirectorv1 "github.com/openstack-k8s-operators/osp-director-operator/api/v1beta1"
+	"github.com/openstack-k8s-operators/osp-director-operator/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type provisionServerConfigOptions struct {
+	Port int
+}
+
+// HttpdConfigMap - custom httpd config map
+func HttpdConfigMap(cr *ospdirectorv1.ProvisionServer) *corev1.ConfigMap {
```

Comment:
see above configmap creation from template comment. this would remove the req for this